project/build.properties

```diff
@@ -1,2 +1,2 @@
 # also update .github/workflows/ci.yml to ensure compatibility of the plugin via scripted
```

Comment:
this note should be taken into account and `.github/workflows/ci.yml` bumped manually, as I was trying to explain in https://github.com/thesamet/sbt-protoc/pull/196#discussion_r517604455

Reply:
Ahh - forgot! Perhaps we should disable Scala steward updates for sbt.

Reply:
or maybe just mergify if that's possible? I thought about it and it should be possible not to duplicate the value here by setting a separate setting in `.github/workflows/ci.yml`, which would set `scriptedSbt` only if the value if not a magic value `latest`. I'll open a PR to try that out.